From d24d382c325c8794c05bcb56b3820b15e4a67e55 Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Tue, 9 Aug 2011 10:42:06 -0400
Subject: [PATCH] macros: Globally add --disable-silent-rules to configure

Various projects have been adding AM_SILENT_RULES from Automake to
their Makefiles for "developer convenience"; the goal being that they
see warnings more easily.

Now really the right way to do this is to have a make wrapper (or an IDE)
that knows how to filter out warnings, but let's leave that aside for now.

But for debugging builds, we really need the full log data.  Being
able to see exactly how e.g. libtool is being run helps a lot for
debugging link problems as an example.
---
 macros |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/macros b/macros
index d7bf415..237e1f4 100644
--- a/macros
+++ b/macros
@@ -35,6 +35,7 @@
   %{_configure} --build=%{_build} --host=%{_host} \\\
 	--program-prefix=%{?_program_prefix} \\\
 	--disable-dependency-tracking \\\
+	--disable-silent-rules \\\
 	--prefix=%{_prefix} \\\
 	--exec-prefix=%{_exec_prefix} \\\
 	--bindir=%{_bindir} \\\
-- 
1.7.6

